src/snmp.c

```diff
@@ -597,6 +599,10 @@ static int csnmp_config_add_host(oconfig_item_t *ci) {
   hd->sess_handle = NULL;
   hd->interval = 0;
 
+  /* These mean that we have not set a timeout or retry value */
+  hd->timeout = 0xFFFFFFFFFFFFFFFF;
```

Comment:
The "invalid" value for `cdtime_t` is zero (`0`), please use that instead.

Reply:
@octo the issue is that the user may set Timeout to 0 in the config (net-snmp default timeout is 1 sec), and I'm trying to decide if I need to set sess->timeout or not. Suggestions?

Reply:
```c
if (hd->timeout != 0) {
  sedd->timeout = CDTIME_T_TO_US(hd->timeout);
}
```

A timeout of zero doesn't make any sense, in that case just leave `sess` unchanged and go with Net-SNMP's default.

Reply:
Sounds good. Thanks.